src/java.base/share/classes/java/lang/invoke/MemberName.java

```diff
@@ -773,7 +773,7 @@ public MemberName getDefinition() {
     }
 
     @Override
-    @SuppressWarnings("deprecation")
+    @SuppressWarnings({"deprecation", "removal"})
```

Comment:
What's going to happen to these warning suppressions when we actually remove (make private) the constructors? Is the intent to just defer that problem until it happens?

Reply:
When the code is changed to not use the constructors, the warning (deprecation and remove) should be removed.
Some of the code is upstream and requires more changes.

Reply:
Yeah, Mandy replied to the question I should have asked: for all the suppressed "removal" points, what is the plan to fix the offending code? If not now, when? ("Later" may be a reasonable answer, but fleshing out the followup tasks would be helpful.)

Reply:
The comment in MemberName makes it clear that `hashCode` method may be called *before* the cache is setup and `valueOf` uses the cache.

Reply:
Its up to the Graal folks to replace their code. In some cases, they are depending on Identity semantics of new Integer().

Reply:
Thanks for the clarification.   Can you file a JBS issue to Graal to make change in its upstream project?